<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotedViewer</class>
 <widget class="QMainWindow" name="NotedViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NotedViewer</string>
  </property>
  <property name="toolTip">
   <string>Categories</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="nbakPath">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>20</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>23</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>NBAK File:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browseButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Select...</string>
    </property>
   </widget>
   <widget class="QListWidget" name="categoriesList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>151</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="notesList">
    <property name="geometry">
     <rect>
      <x>164</x>
      <y>90</y>
      <width>251</width>
      <height>451</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Note List</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="noteText">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>90</y>
      <width>371</width>
      <height>451</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Note Text</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="noteTitle">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>62</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Note Title</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Categories</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Notes</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
